src/data/projects.json

```diff
@@ -1,91 +1,544 @@
+        },
+        "solution": {
+          "title": "What is the solution?",
+          "description": "We contributed to the tokenomics and staking mechanisms to better reward meaningful contributions. A contract was developed using Safe and delegatecall, streamlining Quest actions into a single transaction. A React app was built to manage the airdrop, incorporating security features such as wallet screening, VPN detection, Merkle tree generation, and CAPTCHA verification. We also took charge of launching L3 on Solana, utilizing Wormhole architecture and implementing the first staking contracts."
+        }
+      },
+      "status": "In progress",
+      "links": {
+        "website": [],
+        "twitter": [],
+        "github": []
+      }
+    },
+    {
+      "name": "Safe",
+      "company": {
+        "name": "Safe",
+        "logo": "safe.png"
+      },
+      "tags": ["Architecture Design", "Development"],
+      "content": {
+        "challenge": {
+          "title": "What was the challenge?",
+          "description": "Safe smart wallets on different chains were deployed as separate contracts despite having the same configuration. This causes synchronization issues when changes, such as owner modifications, are made on one chain but not reflected on others, leading to inconsistent security settings across chains. The challenge is to ensure that these configurations are synchronized across chains in a trust-minimized way, especially during security changes."
+        },
+        "solution": {
+          "title": "What is the solution?",
+          "description": "We developed this Safe Liveness module that uses shared cryptographic proofs to verify Safe settings across chains. This module would allow users to batch verify and broadcast changes, ensuring that any updates made on one chain are propagated to all others. So, we will have a master safe, which other deployments will be listening to. The Liveness module can verify Safe ownership based on storage proof and keep everything cross-chain synced."
+        }
+      },
+      "status": "Completed",
+      "links": {
+        "website": [],
+        "twitter": [],
+        "github": ["https://github.com/defi-wonderland/safe-liveness"]
+      }
+    },
+    {
+      "name": "Spells",
+      "company": {
+        "name": "MakerDAO",
+        "logo": "makerdao.png"
+      },
+      "tags": ["Reviewers"],
+      "content": {
+        "challenge": {
+          "title": "What was the challenge?",
+          "description": "MakerDAO is one of the largest and oldest governance in the Crypto DeFi ecosystem and as such has on a daily basis a large number of votes that need to be executed on-chain. These votes which can be payments, parameter changes in the voult, or a contract update are known as spells."
```

Comment:
Can we update the description here?

`"description": "Sky (formerly MakerDAO) is one of the largest and oldest governance systems in DeFi. Every day, it oversees a significant number of on-chain votes that need to be executed. These votes can include payments, parameter adjustments within vaults, or contract updates, collectively referred to as spells."`

Reply:
https://github.com/defi-wonderland/web/pull/114/commits/7255338b31fb438bd055d9d3fac60e947b52b17c